pkgs/applications/editors/moe/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl
+, lzip, ncurses
+}:
+
+with stdenv.lib;
+stdenv.mkDerivation rec{
```

Comment:
Put a space between `rec` and `{` here. (Just for style though I think.)

Reply:
`rec` is reserved word. It is really necessary?

Maybe in a short future we would need to standardize the nix(pkgs) programming style.

Reply:
Oh I'm not sure I just thought this was a typo.